addons/stock/static/src/js/stock_traceability_report_backend.js

```diff
@@ -104,7 +103,10 @@ var stock_report_generic = AbstractAction.extend({
                 url: url_data.replace('output_format', 'pdf'),
                 data: {data: JSON.stringify(dict)},
                 complete: framework.unblockUI,
-                error: crash_manager.rpc_error.bind(crash_manager),
+                error: function () {
+                    var args = ['crash_manager', 'rpc_error'].concat(arguments);
+                    self.call.apply(self, args);
```

Comment:
`self.call('crash_mananager', 'rpc_error', ...arguments);`

(`...` is ES6, which is okay in master since we no longer support IE)